Modules/Map/QuestieMap.lua

```diff
@@ -359,6 +379,21 @@ function QuestieMap:ShowNPC(npcID, icon, scale, title, body, disableShiftToRemov
     data.IsObjectiveNote = false
     data.ManualTooltipData = {}
     data.ManualTooltipData.Title = title or (npc.name .. " (" .. l10n("NPC") .. ")")
+    local isTBC = Questie.IsTBC
+    local isWotlk = Questie.IsWotlk
```

Comment:
It's okay to inline these

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5568/commits/30f291278b358ebd6c31d50da8fce376b3d2dcbd